cache/redis_cache_test.go

```diff
@@ -0,0 +1,76 @@
+package cache
+
+import (
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis/v2"
+	"github.com/go-redis/redis/v8"
+
+	"github.com/contentsquare/chproxy/config"
+)
+
+const cacheTTL = time.Duration(10 * time.Second)
+
+var redisConf = config.Cache{
+	Name: "foobar",
+	Redis: config.RedisCacheConfig{
+		Addresses: []string{"http://localhost:8080"},
+	},
+	Expire: config.Duration(cacheTTL),
+}
+
+func TestCacheSize(t *testing.T) {
+	redisCache := generateRedisClientAndServer(t)
```

Comment:
Isn't it a bit too heavy to spin up a server for each test? I think it could be beneficial to use a single instance of server started and teared down with [TestMain](https://pkg.go.dev/testing#hdr-Main). WDYT?

It's what is done already in `main_test.go` (we can have only one TestMain per package).

Reply:
Indeed! Just checked the CI execution (because those are slow instances) and it looks fine, so yes. Let's leave it like this :)